sklearn/feature_extraction/_dict_vectorizer.py

```diff
@@ -102,18 +103,16 @@ def __init__(self, *, dtype=np.float64, separator="=", sparse=True, sort=True):
         self.sparse = sparse
         self.sort = sort
 
-    def _add_iterable_element(
-        self,
-        f,
-        v,
-        feature_names,
-        vocab,
-        *,
-        fitting=True,
-        transforming=False,
-        indices=None,
-        values=None,
-    ):
+    def _add_iterable_element(self,
+                            f,
+                            v,
+                            feature_names,
+                            vocab,
+                            *,
+                            fitting=True,
+                            transforming=False,
+                            indices=None,
+                            values=None,*awrgs):
```

Comment:
This change is not necessary because `_add_iterable_element` does not make use of a `*awrgs`.